### PR TITLE
Search for creds in workspace when they are not found in defaults

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject

fixes : #921 #929

### CODE REVIEW

File: linchpin/provision/roles/openshift/tasks/provision_resource_group.yml [hunk 1]
```diff
@@ -36,6 +36,17 @@
   no_log: "{{ not debug_mode }}"
   when: res_grp['credentials'] is defined and res_grp['credentials']['filename'] is defined
```
Comment (skatlapa) [inline linchpin/provision/roles/openshift/tasks/provision_resource_group.yml]: what do you think of adding debug task here to check the right workspace is being used:
- debug: msg="{{ lookup('env','workspace') }} is an workspace env variable"

Comment (samvarankashyap) [reply linchpin/provision/roles/openshift/tasks/provision_resource_group.yml]: @skatlapa I tested it on my workspace , I can add that .

Comment (skatlapa) [reply linchpin/provision/roles/openshift/tasks/provision_resource_group.yml]: +1

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**